package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "npm-install-changed",
-  "version": "1.2.1",
+  "version": "1.3.5",
```

Comment:
@adouglas could you remove the version from your PR? I will bump the version after merging.